tokio/src/time/driver/mod.rs

```diff
@@ -99,6 +99,30 @@ pub(crate) struct Driver<T: Park> {
     is_shutdown: bool,
 }
 
+#[cfg(unix)]
+impl std::os::unix::io::AsRawFd for Driver<crate::io::driver::Driver> {
+    fn as_raw_fd(&self) -> std::os::unix::io::RawFd {
+        self.park.as_raw_fd()
+    }
+}
+
+#[cfg(unix)]
+impl std::os::unix::io::AsRawFd for Driver<crate::runtime::io::Driver> {
+    fn as_raw_fd(&self) -> std::os::unix::io::RawFd {
+        use crate::park::Either;
+
+        match self.park {
+            Either::A(ref driver) => {
+                driver.as_raw_fd()
+            }
+            Either::B(_) => {
+                unimplemented!()
+            }
+        }
+        //self.park.as_raw_fd()
```

Comment:
This was left over from the original patch, but probably worth cleaning up now.

Reply:
Yeah was just leaving a comment on this 👍